Scripts/Diagnostics/Azure_VirtualMachine_SI_Enable_Antimalware.ps1

```diff
@@ -0,0 +1,78 @@
+$debug = $true
+
+function Get-MDEPreferences() {
```

Comment:
These commands needs to be run within VM only, and need to check if valid for Linux VMs or not.

Reply:
Added note for the same